public/docs/_examples/server-communication/dart/lib/wiki/wiki_smart_component.dart

```diff
@@ -0,0 +1,36 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'wikipedia_service.dart';
+import 'package:stream_transformers/stream_transformers.dart';
+
+@Component(
+    selector: 'my-wiki-smart',
+    template: '''
+    <h1>Smarter Wikipedia Demo</h1>
```

Comment:
I know it's not done consistently in the ts version, but it's more readable to indent the template content by extra spaces.